pkgs/games/jumpnbump/default.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenv
+, fetchFromGitLab
+, fetchzip
+, SDL2, SDL2_mixer, SDL2_net
+, gtk3, gobject-introspection
+, python3Packages
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "jumpnbump";
+  version = "1.70-dev";
+
+  # By targeting the development version, we can omit the patches Arch uses
+  src = fetchFromGitLab {
+    domain = "gitlab.com";
+    owner = "LibreGames";
+    repo = pname;
+    rev = "b407deef1edcedf9a35dbe9d09b31e797b070bca";
+    sha256 = "03as9k6dw5ms8i6prq53glfsk625npflqh9zzflybnr2ycvhjpkk";
+  };
+
+  data = fetchzip {
+    url = "https://mirandir.pagesperso-orange.fr/files/additional-levels.tar.xz";
```

Comment:
Please put this into a let in.

Reply:
Did so. Could you elaborate a bit (or toss me a pointer) on the advantages and disadvantages of each approach? I've seen both styles in nixpkgs and am unsure of their relative merits.